MFVNeutralino/plugins/VertexHistos.cc

```diff
@@ -408,62 +719,169 @@ void MFVVertexHistos::analyze(const edm::Event& event, const edm::EventSetup&) {
   const double bsz = mevent->bsz;
   const math::XYZPoint bs(bsx, bsy, bsz);
   const math::XYZPoint pv(mevent->pvx, mevent->pvy, mevent->pvz);
-  double caloht = 0.0;
 
   edm::Handle<MFVVertexAuxCollection> auxes;
   event.getByToken(vertex_token, auxes);
 
+
+
   const int nsv = int(auxes->size());
   h_nsv->Fill(nsv, w);
 
-  int ncalojets      = mevent->calo_jet_pt.size();
-  int calojetsptpass = 0;
-  for (int i=0; i < ncalojets; i++) { 
-    if (mevent->calo_jet_pt[i] > 30.0 and fabs(mevent->calo_jet_eta[i]) < 2.5) caloht += mevent->calo_jet_pt[i];
-    if (mevent->calo_jet_pt[i] > 50.0) calojetsptpass++;
-  }
```

Comment:
These look important, are we intentionally dropping them? Do these things just end up somewhere else instead? @pkotamnives @abbywarden @shaunhogan